pkg/agent/containerd/config_linux.go

```diff
@@ -38,6 +38,13 @@ func getContainerdArgs(cfg *config.Node) []string {
 	return args
 }
 
+// TODO: drop once we switch to go1.21 - replace with https://pkg.go.dev/maps#Copy
+func mergeMaps(dst, src map[string]templates.ContainerdRuntimeConfig) {
+	for key, value := range src {
+		dst[key] = value
+	}
+}
+
```

Comment:
Drop this, just have the functions add the entries to the existing map
```suggestion
// TODO: drop once we switch to go1.21 - replace with https://pkg.go.dev/maps#Copy
func mergeMaps(dst, src map[string]templates.ContainerdRuntimeConfig) {
	for key, value := range src {
		dst[key] = value
	}
}

```

Reply:
done